neo/sys/posix/posix_main.cpp

```diff
@@ -61,6 +61,17 @@ If you have questions concerning this license or the applicable additional terms
 #define					MAX_OSPATH 256
 #define					COMMAND_HISTORY 64
 
+
+namespace
+{
+	template<typename T>
+	inline T ignore_result( T x )
```

Comment:
Nevermind. I need to revert this commit without replacement anyway in a later push (see the commit message in the upcoming reversion commit).